lib/mongo/tracing/open_telemetry/command_tracer.rb

```diff
@@ -0,0 +1,320 @@
+        #
+        # @param message [ Mongo::Protocol::Message ] the command message.
+        #
+        # @return [ String | nil ] the collection name, or nil if not applicable.
+        def collection_name(message)
+          case message.documents.first.keys.first
+          when 'getMore'
+            message.documents.first['collection'].to_s
+          when 'listCollections', 'listDatabases', 'commitTransaction', 'abortTransaction'
+            nil
+          else
+            value = message.documents.first.values.first
+            # Return nil if the value is not a string (e.g., for admin commands that have numeric values)
+            value.is_a?(String) ? value : nil
+          end
+        end
+
+        # Extracts the command name from the message.
+        #
+        # @param message [ Mongo::Protocol::Message ] the command message.
+        #
+        # @return [ String ] the command name.
+        def command_name(message)
+          message.documents.first.keys.first.to_s
+        end
+
+        # Extracts the database name from the message.
+        #
+        # @param message [ Mongo::Protocol::Message ] the command message.
+        #
+        # @return [ String ] the database name.
+        def database(message)
+          message.documents.first['$db'].to_s
+        end
+
+        # Checks if query text capture is enabled.
+        #
+        # @return [ Boolean ] true if query text should be captured.
+        def query_text?
+          @query_text_max_length.positive?
+        end
+
+        # Extracts the cursor ID from getMore commands.
+        #
+        # @param message [ Mongo::Protocol::Message ] the command message.
+        #
+        # @return [ Integer | nil ] the cursor ID, or nil if not a getMore command.
+        def cursor_id(message)
+          return unless command_name(message) == 'getMore'
+
+          message.documents.first['getMore'].value
+        end
+
+        # Extracts the logical session ID from the command.
+        #
+        # @param message [ Mongo::Protocol::Message ] the command message.
+        #
+        # @return [ BSON::Binary | nil ] the session ID, or nil if not present.
+        def lsid(message)
+          lsid_doc = message.documents.first['lsid']
+          return unless lsid_doc
+
+          lsid_doc['id'].to_uuid
+        end
+
+        # Extracts the transaction number from the command.
+        #
+        # @param message [ Mongo::Protocol::Message ] the command message.
+        #
+        # @return [ Integer | nil ] the transaction number, or nil if not present.
+        def txn_number(message)
+          txn_num = message.documents.first['txnNumber']
+          return unless txn_num
+
+          txn_num.value
+        end
+
+        # Keys to exclude from query text capture.
+        EXCLUDED_KEYS = %w[lsid $db $clusterTime signature].freeze
+
+        # Ellipsis for truncated query text.
+        ELLIPSES = '...'
```

Comment:
"Ellipsis" (with an 'i') is a single set of three dots, e.g. '...'

"Ellipses" (with an 'e') is the plural, referring to multiple sets of three dots. E.g. "I really like ellipses... I use them everywhere..."

```suggestion
        ELLIPSIS = '...'
```

Reply:
Done